src/frontend/src/app/lobby/lobby.component.ts

```diff
@@ -31,6 +34,7 @@ export class LobbyComponent implements OnInit {
       }, 500);
     } else {
       this.ws.newGameID.subscribe((id) => {
+        this.location.replaceState('/lobby/' + id);
```

Comment:
Why do we need to use replaceState here? Can't we just use the angular router?

Reply:
replaceState doesn't redirect, just changes the url in the browser. Just so the user can copy the link to send to others